data.json

```
@@ -163,6 +163,11 @@
                     "value": "https://ci.lucko.me/view/LuckPerms/"
                 }
             ]
+        },
+            "command": "paste",
+            "title": "PasteBin",
```

"PasteBin" looks stupid in terms of capitalization.
Also, hastebin is preferred over Pastebin as of the reasons BrainStone already mentioned on Discord.
```suggestion
            "title": "Hastebin",
```